packages/electron/README.md

```diff
@@ -0,0 +1,504 @@
+| ---------------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- | ------------------------ |
+| `metricsSubmissionUrl` | String  | Metrics server hostname. By default the value is set to https://events.backtrace.io.                                                                                                                                                                                                                                                                     | `https://events.backtrace.io` | <ul><li>- [ ] </li></ul> |
+| `enable`               | Boolean | Determines if the metrics support is enabled. By default the value is set to true.                                                                                                                                                                                                                                                                       | `true`                        | <ul><li>- [ ] </li></ul> |
+| `autoSendInterval`     | Number  | Indicates how often crash free metrics are sent to Backtrace. The interval is a value in ms. By default, session events are sent on application startup/finish, and every 30 minutes while the application is running. If the value is set to 0. The auto send mode is disabled. In this situation the application needs to maintain send mode manually. | On application startup/finish | <ul><li>- [ ] </li></ul> |
+| `size`                 | Number  | Indicates how many events the metrics storage can store before auto submission.                                                                                                                                                                                                                                                                          | `50`                          | <ul><li>- [ ] </li></ul> |
+
+#### Metrics Usage
+
+```ts
+// metrics will be undefined if not enabled
+client.metrics?.send();
+```
+
+---
+
+### Offline database support
+
+The Backtrace Electron SDK can cache generated reports and crashes to local disk before sending them to Backtrace. This
+is recommended; in certain configurations Electron applications can crash before the SDK finishes submitting data, and
+under slow internet conditions your application might wait in a closing window until the HTTP submission finishes. In
+such an event cached reports will be sent on next application launch.
```

Comment:
It's not true, Crashpad is out of process crash reporting solution so we report reports immediately on crash or when we fail we report it on app restart

Reply:
Changed.